test/python/common.py

```diff
@@ -174,4 +174,31 @@ def _is_ci_fork_pull_request():
     return False
 
 
+def compare_dicts(dict1, dict2, threshold, default=0):
+    """
+    Compare two dictionaries with numeric values.
+
+    Args:
+        dict1 (dict): a dictionary.
+        dict2 (dict): a dictionary.
+        threshold (number): threshold for comparison.
```

Comment:
Random idea: do you think it makes sense to change the `threshold` parameter so it is always between 0 and 1, multiplying it inside the function by `sum(dict1.values())` or similar? It would allow us to have a default value for it (`0.025`?) and save the explicit multiplication during the tests.